examples/panel-basic/README.md

```diff
@@ -16,30 +20,32 @@ Use panel plugins when you want to do things like visualize data returned by dat
    yarn install
```

Comment:
this line must be `npm install`. We should not install dependencies with yarn because we use npm in this repository.